android/build.gradle

```diff
@@ -164,4 +181,9 @@ dependencies {
   androidTestImplementation "androidx.test:core:1.4.0"
   androidTestImplementation 'androidx.test:runner:1.1.0'
   androidTestImplementation "org.mockito:mockito-core:3.+"
+
+  implementation "androidx.compose.ui:ui:1.7.8"
+  implementation "androidx.compose.material:material:1.7.8"
+  implementation "androidx.compose.ui:ui-tooling-preview:1.7.8"
```

Comment:
Is this necessary?

Reply:
Just the one the comment is on.

Reply:
Tested this and looks like we only need

implementation "androidx.compose.ui:ui:1.7.8"

Reply:
We shouldn't need to do this at all. I'll fix in a future stripe-android release. https://github.com/stripe/stripe-android/pull/10721